script/run-nightwatch.sh

```diff
@@ -29,7 +29,7 @@ trap 'if [ $(jobs -p) ] ; then kill $(jobs -p); fi' EXIT
 
 BUILDTYPE=${BUILDTYPE:-vagovdev}
 
-"$(dirname "$0")"/run-mockapi.sh
+"$(dirname "$0")"/run-mockapi.sh &
```

Comment:
adding the `&` because this didn't run asynchronously and would pause. This will make it asynchronously.